beacon-chain/state/interface/phase0.go

```diff
@@ -193,4 +193,5 @@ type WriteOnlyAttestations interface {
 	SetCurrentEpochAttestations(val []*pbp2p.PendingAttestation) error
```

Comment:
Do we also want to delete `SetPreviousEpochAttestations` and `SetCurrentEpochAttestations` from the interface? It seems like it's only used in tests.

Maybe in a different PR is ok too

Reply:
Not sure yet. It can be deleted later as well

Reply:
Approving. Will leave it up to you to decide delete now or later. (Might consider open an issue for delete later)